common/defs/README.md

```diff
@@ -71,19 +73,22 @@ generated from the coin's type and shortcut:
 If a token shortcut has a suffix, such as `CAT (BlockCat)`, the whole thing is part
 of the key (so the key is `erc20:eth:CAT (BlockCat)`).
 
-Sometimes coins end up with duplicate symbols, which in case of ERC20 tokens leads to
-key collisions. We do not allow duplicate symbols in the data, so this doesn't affect
-everyday use (see below). However, for validation purposes, it is sometimes useful
-to work with unfiltered data that includes the duplicates. In such cases, keys are
-deduplicated by adding a counter at end, e.g.: `erc20:eth:SMT:0`, `erc20:eth:SMT:1`.
-Note that the suffix _is not stable_, so these coins can't be reliably uniquely identified.
+Sometimes coins end up with duplicate symbols (keys), which in case of ERC20 tokens leads to
+key collisions. We do not allow duplicate symbols in the built-in data, so this doesn't affect
+everyday use (see below).
+
+External definitions are generated based on data from external APIs on which we rely to have
+the collisions solved.
 
 ## Duplicate Detection
```

Comment:
I think that it's still useful to have everything here as it is, because it describes the process of how we deal with the duplicates - which is still valid. (If what you wanted to say was that I should replace the whole section with just 2 bullet points)